.release_templates/tensorflow1_release_images.yml

```diff
@@ -15,7 +15,7 @@ release_images:
                             # has already been published. Re-released image will have minor version incremented by 1.
     inference:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py36"]
```

Comment:
Why make this change on inference and not on training?

And since this file is actually just an example, why make the change here at all?

Reply:
@saimidu We only have currently py27 and py36 Images for Inferences unlike in Training where it includes py27,py36 and py37.

Reply:
Got it. Thanks.